### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by Darkhost1999**
*Adding FreeBSD and MacOS to about and welcome dialogues.
*Fix multiple instances message dialogue.
Before
FixMe
After
Fixed

<details>
<summary>Original PR</summary>

### Inspired by
Inspiration

#### Changes in this PR

* Remove Default Resolution
Should still be configurable in the config file if you absolutely need this.
* Limit Resolution Scale minimum to 100%
Some themes are effected by this I tried editing my custom theme and it looks like a number is missing  `¯\_(ツ)_/¯`
image
But most themes look like
image

</details>

### CODE REVIEW

**Comment by RainbowCookie32** [comment]
Not sure if removing Resolution altogether from UI is the way to go, some games *can* actually use the mode properly. Maybe moving it to Advanced would be better?

Also, for Resolution Scaling I think it'd be better to limit the slider's minimum value to 100% if you want to avoid users lowering it pointlessly instead of limiting the setting's value everywhere.

**Comment by Darkhost1999** [comment]
> Not sure if removing Resolution altogether from UI is the way to go, some games *can* actually use the mode properly. Maybe moving it to Advanced would be better?

I've seen this argued both ways. Those smart enough to know what games support it are smart enough to edit the config.yml
Smoothies would look for it if moved to advanced tab.

> Also, for Resolution Scaling I think it'd be better to limit the slider's minimum value to 100% if you want to avoid users lowering it pointlessly instead of limiting the setting's value everywhere.

Good point I searched for where to limit that I wasn't seeing one so I did what I know how to.

**Comment by Yahfz** [comment]
Why are removing the option of going under 100% when there's actual value to it? You realize there are iGP with good support in RPCS3 but that just aren't fast enough right? i.e steam deck where it can get bottlenecked by its iGP.

**Comment by Darkhost1999** [comment]
> Why are you removing the option of going under 100% when there's actual value to it? You realize there are iGP with good support in RPCS3 but that just aren't fast enough right? i.e steam deck where it can get bottlenecked by its iGP. 

I wasn't aware of any value for it. I can revert that commit or this entire PR could just be closed if nobody even wants any of it.
The misc fixes can tag team anybodys PR

**Comment by Yahfz** [comment]
The resolution scale behavior shouldn't be changed as there's value for it. As for the removal of the default resolution, I don't agree with it either. I think there's better ways of handling it, such as renaming some of the features. We'd have to come up with a name for resolution scale and default resolution so they understand that they shouldn't change the default resolution if they want to upscale since they sure as hell don't read tooltips.

File: rpcs3/rpcs3qt/about_dialog.ui [hunk 1]
```diff
@@ -124,7 +124,7 @@
...
+           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;RPCS3 is an open-source Sony PlayStation 3 emulator and debugger.&lt;br/&gt;It is written in C++ for Windows, Linux, FreeBSD and Mac funded with &lt;a href=&quot;https://www.patreon.com/Nekotekina&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Patreon&lt;/span&gt;&lt;/a&gt;.&lt;br/&gt;Our developers and contributors are always working hard to ensure this project is the best that it can be.&lt;br/&gt;There are still plenty of implementations to make and optimizations to do.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```
Comment (sampletext32) [inline rpcs3/rpcs3qt/about_dialog.ui:127]: Here you mention OS's, so it should be MacOS, not Mac

Comment (Darkhost1999) [reply rpcs3/rpcs3qt/about_dialog.ui:127]: Thanks

**Comment by Darkhost1999** [comment]
> I think there's better ways of handling it, such as renaming some of the features. We'd have to come up with a name for resolution scale and default resolution so they understand that they shouldn't change the default resolution if they want to upscale since they sure as hell don't read tooltips.

How about simply editing the UI strings like Default Resolution (Do-Not-Edit)
Meanwhile resolution scale I don't have an answer for.
Or
How about moving default resolution to the debug tab instead of advanced? It'll be hidden from normies.
And then we edit the Resolution Scale to just say Resolution. Including in tooltips something like this is resolution scaling not default resolution.

**Comment by RainbowCookie32** [comment]
Sounds too roundabout for something where you can just tell the user to set resolution back to 720p and use scaling instead if the game has issues with other resolutions. 

Nuking the option altogether or moving it to Debug tab just gets on the way of users that actually may want the option.

**Comment by Darkhost1999** [comment]
Can we utilize this in the games list to make an automatic selector
image

Then Default Resolution could turn into a checkbox 1080p mode (or best value)
else 720p

I understand games don't support their "or best values" but how can we fix these issues if we don't accept issue reports using resolutions other than default 720p.
Therefore if we're not going to do anything to fix default resolution then it should take a mini vacation until the feature is both supported and usable for all users.
Or the workaround of moving it to the hidden tab sounds most logical to me just until we actually support the feature for everyone. I want to hear a couple more thoughts I agree full removal is irrational.

**Comment by RainbowCookie32** [comment]
The list of resolutions you see on the game list is what the games report on the param.sfo file iirc, and I don't think it's particularly reliable anyways. Pretty sure I've seen games technically supporting 1080p breaking with it anyways.

**Comment by lextra2** [comment]
Moving the default resolution setting to the config.yml would be a very good change for reasons already named. And having the resolution scale limited to 100% in the UI sounds also like a good idea.

Lets be honest here, the 5 games or whatever that saw any benefit from changing the default resolution can be changed via config. The same is true for resolution scale below 100%. If you run into a gpu bottleneck you can still edit things in the config.

File: rpcs3/rpcs3qt/tooltips.h [hunk 2]
```diff
@@ -152,13 +152,12 @@ class Tooltips : public QObject
...
 		const QString anti_aliasing              = tr("Emulate PS3 multisampling layout.\nCan fix some otherwise difficult to solve graphics glitches.\nLow to moderate performance hit depending on your GPU hardware.");
 		const QString anisotropic_filter         = tr("Higher values increase sharpness of textures on sloped surfaces at the cost of GPU resources.\nModern GPUs can handle this setting just fine, even at 16x.\nKeep this on Automatic if you want to use the original setting used by a real PS3.");
-		const QString resolution_scale           = tr("Scales the game's resolution by the given percentage.\nThe base resolution is always 1280x720.\nSet this value to 100% if you want to use the normal Resolution options.");
+		const QString resolution_scale           = tr("Scales the game's resolution by the given percentage.\nThe base resolution is always 1280x720.\nSet this value to 100% if you want to use the normal Resolution options.\nValues less than 100% will not improve performance");
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/tooltips.h:160]:
```suggestion
		const QString resolution_scale           = tr("Scales the game's resolution by the given percentage.\nThe base resolution is always 1280x720.\nSet this value to 100% if you want to use the normal Resolution options.\nValues less than 100% will not improve performance.");
```

**Comment by lainiwaku** [comment]
only  early  ps3 game doesn't  support 1080p, i think there are also many game, that have 1080p cinematics on the blueray
i never understood why rpcs3 won't let  people  upscale from 1080p
i personally  don't think removing the vhoice of resolution is good idea, we should instead  allow upscale from 1080p  for game  that allow it no ?

**Comment by lextra2** [comment]
> only early ps3 game doesn't support 1080p, i think there are also many game, that have 1080p cinematics on the blueray i never understood why rpcs3 won't let people upscale from 1080p i personally don't think removing the vhoice of resolution is good idea, we should instead allow upscale from 1080p for game that allow it no ?

Like 99% of all PS3 games render at 720p. Having a real PS3 be set to output 1080p means the 720p image is upscaled with bilinear filtering. Videos are 720p

Everything you said is wrong and exactly the reason why this pull request exists.

File: rpcs3/main.cpp [hunk 3]
```diff
@@ -435,7 +435,7 @@ int main(int argc, char** argv)
 		{
 			if (fs::exists(lock_name))
 			{
-				report_fatal_error("Another instance of RPCS3 is running. Close it or kill its process, if necessary.");
+				report_fatal_error("Another instance of RPCS3 is running. \nClose it or kill its process, if necessary.");
```
Comment (Megamouse) [inline rpcs3/main.cpp:438]:
```suggestion
				report_fatal_error("Another instance of RPCS3 is running.\nClose it or kill its process, if necessary.");
```

**Comment by Asinin3** [comment]
I don't like this change. The option shouldn't be removed, it is useful for a number of titles. We have discussed changing this option in staff channels for years and there's a reason we never removed it. PR's with subjective changes like this would be much better if you actually discussed it in our chat platforms beforehand as you'll just end up with a PR that no one wants to merge until changes are made otherwise, and it gets messy.

About the only way to make it more noob-proof without making RPCS3 annoying to use and having to dig through config files is to give users a warning when they change it that can be set to never show again. Or find a better name for the setting or something else. Right now most users know not to touch it but it is a small issue. Putting it in the config file is only going to really annoy everyone else me included.

**Comment by Darkhost1999** [comment]
> PR's with subjective changes like this would be much better if you actually discussed it in our chat platforms beforehand.

I did

Honestly this has been discussed numerous times by others when I didn't chime in and often with me chiming in. It wasn't until the PR was made did I find value in keeping the settings as is. Kind of sucks not having access to "staff channels" because then everybody else is left in the dark. Not even read only channels.

I see my pr just sweeps under the rug someone's devious and fiendish prank until it can be worked on to work like a ps3 does (default resolution) in which I made plans to restore default resolution to the debug tab hidden from normal users so as it can still be used by testers, developers, and regulars of RPCS3. Don't need to now because I also disagree with the PR.

The resolution scaling thing about lower than 100% values I've asked about a lot. I've never gotten a good answer. The good answer is actually from the RPCS3 bot explain saying below 100% will not improve performance. Aka it won't do anything for anyone so why have a value lower than 100?
Yahfz gave a better answer I would've never ever gotten on discord there are igpus and steam decks with low VRAM or other stuff that can benefit from lower b resolution scale.

The other changes I made for miscellaneous fixes can tag team anyone's PR. They don't have to piggy back mine.

The real problem is we need to fix default resolution to work like a PS3 does. This doesn't do that so this PR isn't even fixing anything at all.
Resolution scaling isn't even a problem at all with nothing to fix. Although adding to the tooltip values lower than 100% won't improve performance doesn't hurt anything.

**Comment by Darkhost1999** [comment]
This discussion here has done more in less than 24 hours than any conversation on discord ever has. Tbf

**Comment by jeremybris** [comment]
I like both changes: simplify the user interface for the user... Moving the options to the debug tab would be a good compromise. 

For the people that want to keep it, can you provide specific photo or video evidence that both are needed, not just hypotheticals... The Steam Deck is able to run most games at 720p: https://www.youtube.com/watch?v=fMxVO_eUgEY

There are a lot of old settings I think should also be moved to the debug tab, like very few people use the interpreters practically, in the CPU tab.

BTW, fixes #11569

**Comment by kd-11** [comment]
> I like both changes: simplify the user interface for the user... Moving the options to the debug tab would be a good compromise.
> 
> For the people that want to keep it, can you provide specific photo or video evidence that both/either are needed, not just hypotheticals... The Steam Deck runs most games at 720p: https://www.youtube.com/watch?v=fMxVO_eUgEY
> 
> There are a lot of old settings I think should also be moved to the debug tab, like very few people use the interpreters practically, in the CPU tab.
> 
> BTW, fixes #11569

Compared to most IGPs on the market the steam deck is like a 3090ti. Also the steam deck has 100% gpu load from test results provided to us. Letting resolution scale less than 100 is therefore not going anywhere. Its also not confusing, like every game has this setting.

**Comment by kd-11** [comment]
As for native 1080p support - back in the PS3 gen, higher resolution meant cutting features. So 1080p mode in the handful of titles that truly support it internally, often has inferior visuals (removed shadows, simplified world detail, removed postprocessing) because the console could not handle the extra load. The results are far better at 720p and a minor upscale of 50% gets you 1080p with all the bells and whistles intact.

File: rpcs3/rpcs3qt/tooltips.h [hunk 4]
```diff
@@ -158,7 +158,7 @@ class Tooltips : public QObject
...
 		const QString anti_aliasing              = tr("Emulate PS3 multisampling layout.\nCan fix some otherwise difficult to solve graphics glitches.\nLow to moderate performance hit depending on your GPU hardware.");
 		const QString anisotropic_filter         = tr("Higher values increase sharpness of textures on sloped surfaces at the cost of GPU resources.\nModern GPUs can handle this setting just fine, even at 16x.\nKeep this on Automatic if you want to use the original setting used by a real PS3.");
-		const QString resolution_scale           = tr("Scales the game's resolution by the given percentage.\nThe base resolution is always 1280x720.\nSet this value to 100% if you want to use the normal Resolution options.");
+		const QString resolution_scale           = tr("Scales the game's resolution by the given percentage.\nThe base resolution is always 1280x720.\nSet this value to 100% if you want to use the normal Resolution options.\nValues less than 100% will not improve performance.");
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/tooltips.h:161]: Maybe like this:
```suggestion
		const QString resolution_scale           = tr("Scales the game's resolution by the given percentage.\nThe base resolution is always 1280x720.\nSet this value to 100% if you want to use the normal Resolution options.\nValues below 100% will usually not improve performance.");
```

**Pull request merged**